bandcamp_dl/bandcamp_dl.py

```diff
@@ -33,21 +31,26 @@
     http://twitter.com/miniarray
     http://github.com/miniarray
 
+Anthony Forsberg:
+    http://evolution0.github.io
+    http://github.com/evolution0
+
 Iheanyi:
     Feel free to use this in any way you wish. I made this just for fun.
-    Shout out to darkf for writing a helper function for parsing the JavaScript! """
-
+    Shout out to darkf for writing a helper function for parsing the JavaScript!
+"""
 
-from docopt import docopt
-from Bandcamp import Bandcamp
-from BandcampDownloader import BandcampDownloader
 import os
+from docopt import docopt
+from Bandcamp_Parse import Bandcamp
+from Bandcamp_Download import BandcampDownloader
```

Comment:
Mainly for clarity.

`from Bandcamp import Bandcamp` you have to admit tells you nothing when reading the code, now at first glance you know that you are importing a class related to parsing data from bandcamp.

As for Bandcamp_Download that was mainly for consistency and the fact that importing classes from modules of the same exact name seems a little odd. Though I later learned this is common (though with lower cased files, upper cased classes)

If you would like it changed back that is cool but I would recommend using the lowercase filename/uppercase class scheme.